### PR TITLE
Upgrade joern version to 1.1.1456

### PR DESCRIPTION
**Pull request by dbMundada**

### CODE REVIEW

File: src/main/scala/ai/privado/languageEngine/java/tagger/sink/JavaAPITagger.scala [hunk 1]
```diff
@@ -154,7 +154,7 @@ class JavaAPITagger(cpg: Cpg) extends ForkJoinParallelCpgPass[RuleInfo](cpg) {
     if (apis.nonEmpty && filteredLiteralSourceNode.nonEmpty) {
       val apiFlows = apis.reachableByFlows(filteredLiteralSourceNode).l
       apiFlows.foreach(flow => {
-        val literalCode = flow.elements.head.originalPropertyValue.getOrElse(flow.elements.head.code)
+        val literalCode = flow.elements.head.code
```
Comment (khemrajrathore) [inline src/main/scala/ai/privado/languageEngine/java/tagger/sink/JavaAPITagger.scala:157]: @dbMundada don't change anything here,  instead change CfgNode to AstNode for the travesal originalTraversalValue which is presemt in privado language module

Comment (khemrajrathore) [reply src/main/scala/ai/privado/languageEngine/java/tagger/sink/JavaAPITagger.scala:157]: For reference you can refer the PR raised by Rahul

Comment (khemrajrathore) [reply src/main/scala/ai/privado/languageEngine/java/tagger/sink/JavaAPITagger.scala:157]: Similar commnet for 2 other places below

**Review by pandurangpatil — APPROVED** [review]
looks good

**Pull request merged**